packages/image_picker/image_picker_platform_interface/lib/src/platform_interface/image_picker_platform.dart

```diff
@@ -61,11 +61,11 @@ abstract class ImagePickerPlatform extends PlatformInterface {
   /// In Android, the MainActivity can be destroyed for various reasons. If that happens, the result will be lost
   /// in this call. You can then call [retrieveLostDataAsDartIoFile] when your app relaunches to retrieve the lost data.
   @Deprecated('Use pickImage instead.')
-  Future<String> pickImagePath({
-    @required ImageSource source,
-    double maxWidth,
-    double maxHeight,
-    int imageQuality,
+  Future<String?> pickImagePath({
```

Comment:
The docs for this method does not specify what to do when the future returns `null`, or what it means.